cpprevolve/revolve/gazebo/plugin/RobotController.cpp

```diff
@@ -200,25 +207,39 @@ void RobotController::LoadBrain(const sdf::ElementPtr _sdf)
     return;
   }
 
-  auto brain = _sdf->GetElement("rv:brain");
-  auto controller = brain->GetElement("rv:controller")->GetAttribute("type")->GetAsString();
-  auto learner = brain->GetElement("rv:learner")->GetAttribute("type")->GetAsString();
-  std::cout << "Loading controller " << controller << " and learner " << learner << std::endl;
+  auto brain_sdf = _sdf->GetElement("rv:brain");
```

Comment:
also here we should double check if there are errors in the SDF and report problems instead of waiting for a segfault event.